libfwupdplugin/fu-usb-device.c

```diff
@@ -386,7 +421,7 @@ fu_usb_device_open(FuDevice *device, GError **error)
 		FuUsbDeviceInterface *iface = g_ptr_array_index(priv->interfaces, i);
 		if (!fu_device_retry_full(device,
 					  fu_usb_device_claim_interface_cb,
-					  FU_USB_DEVICE_CLAIM_INTERFACE_RETRIES,
+					  priv->claim_retry_count,
```

Comment:
I didn't include a check for `claim_retry_count > 0` here, I just assumed that a value of zero would cause this to run a single time. Is that a fair assumption? Should it be 1 instead of 0?

Reply:
I think that's a dangerous assumption -- could we do the same as we did for open? i.e. don't call fu_device_retry_full() if the claim_retry_count is zero.

Reply:
Sure, can do. Am I calling the callback directly then? Is there precedent for that elsewhere in the codebase?

Reply:
Nah, just call g_usb_device_claim_interface() directly.